README.md

```diff
@@ -27,6 +51,52 @@ markets = ["DE", "PL"]
 | DBL  | Double      | P2:3 A2:2 C0:1 |
 ```
 
+### Basic section with possible field types
+
+```ini
+[CONTRACT]
+country = "Poland"                  // String
+markets = ["PL", "DE", "UK"]        // Array
+75042 = {                           // Dictionary
+    view = "SV"                     // String
+    loc  = ["M", "B"]               // Array
+    dist = { beach_km = 4.1 }       // Dictionary
+}
+```
+
+### Complex document built from few sections
+
+```ini
+[CONTRACT]
+country = "Poland"
+markets = ["PL", "DE", "UK"]
+75042 = {
+    view = "SV"
+    loc  = ["M", "B"]
+    dist = { beach_km = 4.1 }
+}
+
+[RATE.PLAN]
+|       dates       | code |  description   |    rooms    | rules |
```

Comment:
shouldn't we have the separator here as well? 🤔 (i.e. `|----|...`)

Reply:
thanks, applied